metaci/release/utils.py

```diff
@@ -124,6 +129,32 @@ def send_release_webhook(project_config, release, config_item=None):
         raise Exception("\n".join(err["message"] for err in result["errors"]))
 
 
+def send_submit_webhook(release, config_item=None):
+    if (
+        release is None
+        or not settings.METACI_RELEASE_WEBHOOK_URL
+        or not settings.METACI_START_STOP_WEBHOOK
+        or not config_item
+    ):
+        return
+    logger.info(
+        f"Sending submit webhook for {release} to {settings.METACI_RELEASE_WEBHOOK_URL}"
+    )
+
+    payload = {"case_id": release.change_case_link}
+    token = jwt_for_webhook()
+    response = requests.post(
+        f"{settings.METACI_RELEASE_WEBHOOK_URL}/case/{release.change_case_link}/submit",
+        json=payload,
+        headers={"Authorization": f"Bearer {token}"},
+    )
+    result = response.json()
+    if result["success"]:
+        return
+    else:
+        raise Exception("\n".join(err["message"] for err in result["errors"]))
+
```

Comment:
send_submit_webhook doesn't have any test coverage

Reply:
That's because gus sandbox configuration doesn't support at the moment so it's blocked. Was going to write test coverage once we know the proper response payload.

Reply:
@jofsky ok, fair enough.

Reply:
@jofsky I don't think the actual response from GUS matters that much here. We just want to know whether it succeeded or failed. And errors should be reported by gus-bus in the same way as for other endpoints. So I think we could go ahead and write a test and probably not need to change it much later.

Reply:
Will do now.